contracts/core/src/xchain/FeeOracleV2.sol

```diff
@@ -14,30 +14,35 @@ contract FeeOracleV2 is IFeeOracle, IFeeOracleV2, OwnableUpgradeable {
     /**
      * @notice Base protocol fee for each xmsg.
      */
-    uint72 public protocolFee;
+    uint96 public protocolFee;
 
     /**
-     * @notice Base gas limit for each xmsg.
+     * @notice Address allowed to set gas prices and to-native conversion rates.
      */
-    uint24 public baseGasLimit;
+    address public manager;
 
     /**
-     * @notice Address allowed to set gas prices and to-native conversion rates.
+     * @notice Conversion rate from this chain's gas token to another native token, by gas token ID.
      */
-    address public manager;
+    mapping(uint8 gasToken => uint256) public tokenToNativeRate;
```

Comment:
Converion rate from `gasToken` to this chain's native token (normalized by CONVERSION_RATE_DENOM)